hack/jenkins/e2e-runner.sh

```diff
@@ -113,7 +113,7 @@ function get_latest_trusty_image() {
     # Retry the gsutil command a couple times to mitigate the effect of
     # transient server errors.
     for n in $(seq 3); do
-      image="$(gsutil cat "gs://${image_project}/image-indices/latest-test-image-${image_index}")" && break || sleep 1
+      image="$(gsutil cat "gs://${image_project}/image-indices/latest-base-image-${image_index}")" && break || sleep 1
```

Comment:
As of now yes as it's been using the old product name that we don't want to mention in public. We'll make another one that's okay to share.